### PR TITLE
fix: 🐛 activity creation redirect and link spacing issues

### PR DESCRIPTION
**Pull request by blp100**
## Description

- fix issue where page does not scroll to top after `redirect`
- reduce link spacing at sidebar to improve click accuracy

### CODE REVIEW

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const ActivityCreationSideBar = ({
       <H1>新增活動</H1>
       <div className="space-y-6">
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#organizer-setting"
```
Comment (ellallu0903) [inline src/components/ActivityCreation/ActivityCreationSideBar.tsx:54]:
```suggestion
          href="#organizer"
```

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ const ActivityCreationSideBar = ({
           <LinkIcon className="mt-1 size-4 opacity-0 transition-opacity duration-300 group-hover:opacity-100" />
         </Link>
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#activity-setting"
```
Comment (ellallu0903) [inline src/components/ActivityCreation/ActivityCreationSideBar.tsx:63]:
```suggestion
          href="#activity"
```

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx [hunk 3]
```diff
@@ -79,7 +79,7 @@ const ActivityCreationSideBar = ({
           })}
         </div>
         <Link
-          className="group flex items-center gap-2 pl-6"
+          className="group flex w-fit items-center gap-2 pl-6"
           href="#ticket-setting"
```
Comment (ellallu0903) [inline src/components/ActivityCreation/ActivityCreationSideBar.tsx:83]:
```suggestion
          href="#tickets"
```

**Review by ellallu0903 — COMMENTED** [review]
建立活動完跳轉的部分測試沒問題，錨點的部分 id 好像沒有對應到。

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx (hunk 3 above)
Comment (blp100) [reply src/components/ActivityCreation/ActivityCreationSideBar.tsx]: fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx (hunk 2 above)
Comment (blp100) [reply src/components/ActivityCreation/ActivityCreationSideBar.tsx]: fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010

File: src/components/ActivityCreation/ActivityCreationSideBar.tsx (hunk 1 above)
Comment (blp100) [reply src/components/ActivityCreation/ActivityCreationSideBar.tsx]: fix it : 6b854e58eccccd4c67c3c6f573ec6b97a0395010

**Review by ellallu0903 — APPROVED** [review]

**Pull request merged**